Lecture2/Task2/Student.java

```diff
@@ -0,0 +1,48 @@
+public class Student implements Comparable<Student> {
+
+	private int id, age;
```

Comment:
all field - final, as it init only in constructor

Reply:
done